source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnRenderer.mm

```diff
@@ -40,15 +41,20 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     ACRColumnView* column = [[ACRColumnView alloc] initWithStyle:(ACRContainerStyle)columnElem->GetStyle()
                                                      parentStyle:[viewGroup style] hostConfig:acoConfig superview:viewGroup];
 
+    auto backgroundImage = columnElem->GetBackgroundImage();
```

Comment:
can you put line 44 & 47 in renderBackgroundImage()? and change renderBackgroundImage() to take shared_ptr<CollectionTypeElement> here and in ContainerRenderer?